.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-  - "0.10"
\ No newline at end of file
+  - "0.12"
```

Comment:
I specified in the `package.json` that lib works with Node 10 and higher. I would assume that tests should be ran on 0.10, 0.11 and 0.12 node versions

http://docs.travis-ci.com/user/languages/javascript-with-nodejs/